zbus/src/connection/builder.rs

```diff
@@ -85,6 +90,36 @@ impl<'a> Builder<'a> {
         Ok(Self::new(Target::Address(Address::system()?)))
     }
 
+    /// Create a builder for the [IBus] daemon.
+    ///
+    /// [IBus]: https://en.wikipedia.org/wiki/Intelligent_Input_Bus
+    #[cfg(feature = "ibus")]
+    pub async fn ibus() -> Result<Self> {
+        let child_process = Command::new("ibus")
+            .args(["address"])
+            .stdout(Stdio::piped())
+            .spawn()
+            .expect("Fail to call `ibus address`");
+
+        #[cfg(not(feature = "tokio"))]
+        let output = child_process
+            .output()
+            .await
+            .expect("Fail to run `ibus address`");
+
+        #[cfg(feature = "tokio")]
+        let output = child_process
+            .wait_with_output()
+            .await
+            .expect("Fail to run `ibus address`");
```

Comment:
Sorry, just noticed this part. Why do they need to be different? `tokio::process::Command` also has `output` method, which should be good enough for us?

Actually, `abstractions::process::Command::output` can do most stuff already for us here? :thinking: You don't even need to bother with piping etc.